tests/lib/rules/no-at-ember-render-modifiers.js

```diff
@@ -0,0 +1,54 @@
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-at-ember-render-modifiers');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+});
+ruleTester.run('no-at-ember-render-modifiers', rule, {
+  valid: ['import x from "x"', ''],
```

Comment:
Let's include more valid:
* `import { x } from 'foo';`
* `import x from '@ember/foo';`
* `import x from '@ember/render-modifiers-foo';`

Reply:
oh I see the reason behind your suggestion above now -- seems .. idk, edge casey?

Reply:
It's trivial to handle this.

Reply:
updated!